README.md

```diff
@@ -21,7 +21,7 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 <a href="docs/images/snippets.md#quicksort">
 <!--
 Edit snippet in docs/images/snippets.md and:
-https://drive.google.com/drive/folders/1-rsUjiya7dSZ87L8kpZmu3MZghRVxzLA
+https://docs.google.com/drawings/d/1uQd1pfdXzxzxrzCm30gdMEQO3dHzJn13qRG8JQentbg/edit?usp=sharing&resourcekey=0-S8Y8V69A1gEblQWINbAYaw
```

Comment:
I'm uncertain which snippets file this was referring to

Reply:
```suggestion
https://drive.google.com/drive/folders/1QrBXiy_X74YsOueeC0IYlgyolWIhvusB
```

Looks like the link was fixed in other examples but not here.

Reply:
Thanks. I've updated the snippets link